File: lib/easy/distribute_candies.dart
int distributeCandies(List<int> candyType) {
  int stop = candyType.length ~/ 2;
  Set<int> unique = {};
  for (int candy in candyType) {
    if (unique.length < stop) {
      unique.add(candy);
    } else {
      return stop;
    }
  }
  return unique.length;
}

File: src/easy/DistributeCandies.swift
func distributeCandies(_ candyType: [Int]) -> Int {
    let stop = candyType.count / 2
    var unique = Set<Int>()
    for candy in candyType {
        if unique.count < stop {
        unique.insert(candy)
        }
        else { return stop }
    }
    return unique.count
}
